internal/schema/schema.go

```diff
@@ -291,6 +291,11 @@ func (s *Schema) Parse(schemaString string, useStringDescriptions bool) error {
 			if !ok {
 				return errors.Errorf("type %q is not an interface", intfName)
 			}
+			for _, f := range intf.Fields.Names() {
+				if obj.Fields.Get(f) == nil {
+					return errors.Errorf("interface %q expects field %q but %q does not provide it", intfName, f, obj.Name)
```

Comment:
All 3 variables (`intfName`, `f` and `obj.Name`) are of type `string`... can you use `%s` in your formatting instead of `%q`. Otherwise it's confusing.

Reply:
`%q` is used for quoting same as above err formatting line 292?